documentation/concepts/03A_transaction_fees.md

```diff
@@ -0,0 +1,78 @@
+---
+id: transactions
+title: Transactions Fees
+sidebar_label: Transaction Fees
+---
+
+# Transactions Fees
+
+A **transaction fee** is a fee that is required in order to process a transaction on the Aleo network. Currently, there are three types of transaction fees that correspond to each transaction type: **deploy** and **execute**. Transaction sizes are averages using [Big _O_ notation (on the order of)](https://en.wikipedia.org/wiki/Big_O_notation).
+
+## Aleo Credits Demonination Table
+
+|Size of Transaction|Denomination|Value|Example|
+|:-:|:-:|:-:|:-:|
+|Byte|microcredit|1|Lowest denomination|
+|Kilobyte (KB)|millicredit|1000 microcredits|Most transaction types average between 3 and 10 millicredits|
+|Megabyte (MB)|credit|1000 millicredits|Assuming a 1:1 USD mapping|
+
+# Types of Transaction Fees
+
+## Execute Transaction Fee
+A fee incurred when executing an application on the Aleo blockchain
+
+### Structure (optimistic)
+
+|Part of Execute Transaction|Size| Multiplier|Cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction wrapper`|0 KB|1|0 millicredits|
+|`Transition wrapper`|0 KB|1|0 millicredits|
+|`Input`|1 KB|2|2 millicredits|
+|`Output`|1 KB|2|2 millicredits|
+|`Proof`|1 KB|1|1 millicredit|
+
+|Total|Size|Multiplier|Total cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction -> Transition`|5 KB|1|5 millicredits|
+
+### Structure (pessimistic)
+
+|Part of Execute Transaction|Size| Multiplier|Cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction wrapper`|1 KB|65|65 millicredits|
+|`Transition wrapper`|1 MB|32|32 credits|
+|`Input`|1 KB|16 * 32|512 millicredits|
+|`Output`|1 KB| 16 * 32|512 millicredits|
+|`Proof`|1 KB|1|1 millicredit|
+
+|Total|Size|Multiplier|Total cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction -> Transition`|32 MB|1|32 credits|
+
+Notes
+* The optimistic execute transaction fee may increase to prevent spamming; it is currently stable between 3 and 5 millicredits
```

Comment:
Maybe add: "it is currently stable between 3 and 5 millicredits for a basic `credits.aleo` transfer`